benchmarks/src/main/scala/scalaz/effect/IODeepAttemptBenchmark.scala

```diff
@@ -28,16 +28,14 @@ class IODeepAttemptBenchmark {
 
   @Benchmark
   def futureDeepAttempt(): BigInt = {
-    import scala.util.{Success, Failure}
+    import scala.util.Success
     import scala.concurrent.Future
     import scala.concurrent.duration.Duration.Inf
 
     def descend(n: Int): Future[BigInt] =
-      if (n == depth) Future.failed(new Error("Oh noes!"))
```

Comment:
It could easily be argued that throwing Errors and expecting to catch them is madness, that is unless of course, you prefer writing unreasonable applications:

«An Error is a subclass of Throwable that indicates serious problems that a reasonable application should not try to catch.» - https://docs.oracle.com/javase/7/docs/api/java/lang/Error.html

Reply:
All the other implementations support throwing and catching errors. `Future` is the odd duck out.

I can special case this for `Future`, as I have done for numerous other things (wrapping of `Error`, for example).